config/01-ce-collector-audit-payload.conf

```diff
@@ -0,0 +1,24 @@
+# Settings for the collector audit payloads function.
+# The easiest way to direct the payload logging information here is to
+# set the environment variable CONDORCE_COLLECTOR_HOST on worker nodes
+# pointing to this condor-ce host.  Then pilot jobs that use condor
+# will automatically forward their payload classads here for auditing.
+
+# You can edit this file but that prevents future rpm upgrade
+# replacements, so it is better to override these settings in 
+# /etc/condor-ce/99-ce-collector-audit-payload.conf
+
+# Set this value to enable the plugin
+# COLLECTOR_PLUGIN_PYTHON_MODULES = htcondorce.audit_payloads
+
+# Set these values to enable the log file
+# COLLECTOR_AUDIT_LOG = $(LOG)/AuditPayloadLog
+# COLLECTOR_DEBUG = D_AUDIT, $(COLLECTOR_DEBUG)
```

Comment:
Wait, by all of the above did you mean to include setting COLLECTOR_PLUGIN_PYTHON_MODULES = htcondorce.audit_payloads to enable the plugin by default, or just enable the log file?